rustls/src/verify.rs

```diff
@@ -634,8 +650,12 @@ fn pki_error(error: webpki::Error) -> Error {
 pub struct NoClientAuth;
 
 impl NoClientAuth {
-    /// Constructs a `NoClientAuth` and wraps it in an `Arc`.
-    pub fn new() -> Arc<dyn ClientCertVerifier> {
+    /// Constructs a [`NoClientAuth`], wraps it in an [`Arc`] and coerces it to
```

Comment:
This documentation was already using the third person while the other two were using the imperative mood. Should I change all to third person or the other way around ?

Reply:
Let's bring this one in line with the others.